collect_sources.sh

```diff
@@ -47,17 +47,13 @@ fetch_deb_src() {
 # exclude binary-only Raspbian packages
 EXCLUDE='realvnc-vnc|oracle-java8-jdk'
```

Comment:
These two are not part of raspbian/raspios anymore. At least I can't find them.

Reply:
realvnc-vnc is source of package: realvnc-vnc-server, it is kept
the oracle-java8-jdk removed